database/models/games.js

```diff
@@ -1,14 +0,0 @@
-const mongoose = require('mongoose');
-
-const gameSchema = new mongoose.Schema({
-  bundle_id: { type: Number, unique: true },
-  game_info: [
-    {
-      title: String,
-      publisher: String
-    }
-  ]
-})
-
-const Game = mongoose.model('Game', gameSchema);
-module.exports = Game;
```

Comment:
I removed this entirely but kept the file incase I get around to adding Game bonuses